src/v2/services/test-data.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const addressesRepo = require('../connectors/repository/addresses');
+const companiesRepo = require('../connectors/repository/companies');
+const companyAddressesRepo = require('../connectors/repository/company-addresses');
+const companyContactsRepo = require('../connectors/repository/company-contacts');
+const contactsRepo = require('../connectors/repository/contacts');
+const documentsRepo = require('../connectors/repository/documents');
+const documentRolesRepo = require('../connectors/repository/document-roles');
+const invoiceAccountAddressesRepo = require('../connectors/repository/invoice-account-addresses');
+const invoiceAccountsRepo = require('../connectors/repository/invoice-accounts');
+
+/**
+ * Calls through to the relevant repositories to delete all test
+ * data that may have been created.
+ */
+const deleteAllTestData = () => {
+  return Promise.all([
```

Comment:
Is there a risk these could run out of order and, e.g. try to delete document before related tables are deleted?

Reply:
Thanks for this, quite correct. I don't know how I got away with this in testing. Probably time for a change in career. 🤦